content/download/fedora.adoc

```diff
@@ -28,16 +38,5 @@ dnf install kicad
 If you don't have copr install with:
 
 ----
-yum install dnf-plugins-core
+dnf install dnf-plugins-core
 ----
-
-As a side note, the wxwidgets build in the copr build is forcing `--with=gtk2` on wxgtk, otherwise eeschema and the legacy canvas does not work properly. 
```

Comment:
Uh, oh, wow. The COPR build seems to have diverged a bit more than I thought:
http://copr-dist-git.fedorainfracloud.org/cgit/@kicad/kicad/kicad.git/tree/kicad.spec

Nevertheless; the important bit is:

`   BuildRequires:  compat-wxGTK3-gtk2-devel`